dashboard/app/models/levels/blockly.rb

```diff
@@ -807,7 +810,8 @@ def localize_behaviors(block_xml)
       end
     end
 
-    block_xml.xpath(".//#{tag}[@name=\"VAR\"]").each do |parameter|
+    # Explicitly check for title tags even if they are not the desired type.
+    block_xml.xpath(".//#{tag}|//title[@name=\"VAR\"]").each do |parameter|
```

Comment:
Naive question: is `title` effectively deprecated in favor of `field`?  If so, why still support it?